src/applications/personalization/view-dependents/components/ViewDependentsList/ViewDependentsListItem.jsx

```diff
@@ -49,25 +51,28 @@ function ViewDependentsListItem(props) {
         ) : null}
       </dl>
       {manageDependentsToggle && (
-        <button
-          onClick={handleClick}
-          className="usa-button-secondary vads-u-margin-x--2 vads-u-background-color--white"
-        >
-          Remove this dependent
-        </button>
-      )}
-      <div
-        className={`vads-l-row vads-l-row vads-u-padding-x--2 vads-u-display--${
-          open ? 'flex' : 'none'
-        }`}
-      >
-        <div className="vads-l-col--8">
-          <p>
-            To remove this dependent from your VA benefits, please enter the
-            information below.
-          </p>
+        <div>
+          <button
+            onClick={handleClick}
+            className="usa-button-secondary vads-u-margin-x--2 vads-u-background-color--white"
+          >
+            Remove this dependent
+          </button>
+          <div
+            className={`vads-l-row vads-l-row vads-u-padding-x--2 vads-u-display--${
+              open ? 'flex' : 'none'
+            }`}
+          >
+            <div className="vads-l-col--8">
```

Comment:
May want to do `className="vads-l-col--12 medium-screen:vads-l-col--8"` to get the full width of the viewport on mobile

Reply:
We havent seen a finished mockup for this, its still going through reviews, so I will hold off on this.